Doc/whatsnew/3.14.rst

```diff
@@ -295,6 +295,19 @@ For further information on how to build Python, see
 
    __ https://en.wikipedia.org/wiki/Tail_call
 
+.. attention::
+
+    This section previously reported a 9-15% geomean speedup. This number has since been
+    cautiously revised down to 3-5%. While we expect performance results to be better
+    than what we report, our estimates are more conservative due to a
+    `compiler bug <https://github.com/llvm/llvm-project/issues/106846>`_ found in
+    Clang/LLVM 19. We were unaware of this bug, and it artifically boosted
```

Comment:
Instead of saying "artificially boosted our numbers" (which seems to imply it magically made tail calling even better), maybe say "which negatively impacted the performance of the traditional non-tail calling interpreter loop"

Reply:
Yeah Stefan Marr pointed out that the artificially boosting comment makes no sense too. So I addressed that here https://github.com/python/cpython/pull/130911